README.md

```diff
@@ -80,6 +105,8 @@ interceptor.use((req) => {
 
 ##### Response mocking
 
+When a request middleware returns a [`MockedResponse`](#MockedResponse) object, it's being used as a mocked response to the intercepted request. This library aligns an abstract mocked response interface to the actual response in correspondence to the module that has issued the request.
```

Comment:
```suggestion
When a request interceptor middleware returns a [`MockedResponse`](#MockedResponse) object, it will then be returned as the mocked response to the intercepted request. This library maps an abstract mocked response interface to the expected response of the module that issued the request.
```

Reply:
The second sentence might not be needed here as it's a little confusing to read, no matter which way it's written.

Reply:
The point of the second sentence is to explain that NRI uses an abstract response interface and then automatically creates a proper response object from it, depending on what module has issued the request (http/xhr). That's useful to highlight, because different request issuing modules create and expect a different response shape.

Reply:
That makes sense, thanks for the additional clarification! I don't know why I had so much trouble reading that line.